test/integration/resultsPagePlace.js

```diff
@@ -90,23 +90,13 @@ describe('The place results page', () => {
       });
   });
 
-  function expectSearchAgainPage($) {
-    expect($('.error-summary-heading').text())
-      .to.contain('You must insert a place or a postcode to find a pharmacy.');
-
-    expect($('.link-back').text()).to.equal('Back');
-    // eslint-disable-next-line no-script-url
-    expect($('.link-back').attr('href')).to.equal('javascript:history.back();');
-  }
-
   it('should return search page for empty search', (done) => {
     chai.request(server)
       .get(resultsRoute)
       .query({ location: '' })
       .end((err, res) => {
         iExpect.htmlWith200Status(err, res);
         const $ = cheerio.load(res.text);
-        expectSearchAgainPage($);
```

Comment:
Checking the contents of the error text is still worthwhile. Can you add back the check https://github.com/nhsuk/connecting-to-services/pull/378/files#diff-92428b787dba582ae2e87d8a8fabc1e6L94 please

Reply:
Done